packages/embark-debugger/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "embark-debugger",
+  "version": "4.1.0-beta.0",
+  "author": "Iuri Matias <iuri.matias@gmail.com>",
+  "contributors": [],
+  "description": "Debugger component for Embark",
```

Comment:
This description should be copied into the README.

Reply:
Done.